.pre-commit-config.yaml

```diff
@@ -97,18 +97,39 @@ repos:
           - types-tqdm
           - types-flask
 
+  # Check for spelling
+  - repo: https://github.com/crate-ci/typos
+    rev: v1.23.6
+    hooks:
+      - id: typos
+        exclude: .html
+
   # Catch common capitalization mistakes
   - repo: local
     hooks:
       - id: disallow-caps
         name: Disallow improper capitalization
         language: pygrep
         entry: PyBind|Numpy|Cmake|CCache|Github|PyTest|Mqt|Tum
-        exclude: .pre-commit-config.yaml
+        exclude: \.pre-commit-config\.yaml|\.joblib$
```

Comment:
Two questions on this one:
- Why the sudden escapes of the dots, and
- Why the additional `.joblib` exclude that was not necessary before

Reply:
I reverted these changes.